pygmo_plugins_nonfree/core.cpp

```diff
@@ -1,50 +1,108 @@
-#include <boost/python/docstring_options.hpp>
-#include <boost/python/module.hpp>
+#include <pybind11/numpy.h>
+#include <pybind11/pybind11.h>
+#include <boost/archive/text_oarchive.hpp>
+#include <boost/numeric/conversion/cast.hpp> 
 
-#include <pygmo/algorithm_exposition_suite.hpp>
-#include <pygmo/register_ap.hpp>
+#include <iostream>
```

Comment:
A few wrong/missing includes here: ``sstream``, ``pagmo/s11n.hpp`` are missing, ``<boost/archive/text_oarchive.hpp>`` not necessary as all the serialization bits are in ``pagmo/s11n.hpp``